sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/implementation/encryption/CosmosEncryptor.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.encryption;
+
+import com.azure.cosmos.implementation.encryption.api.DataEncryptionKey;
+import com.azure.cosmos.implementation.encryption.api.DataEncryptionKeyProvider;
+import com.azure.cosmos.implementation.guava25.base.Preconditions;
+
+/**
+ * Provides the default implementation for client-side encryption for Cosmos DB.
+ * See https://aka.ms/CosmosClientEncryption for more information on client-side encryption support in Azure Cosmos DB.
```

Comment:
By the way - this url doesn't work - https://aka.ms/CosmosClientEncryption

Reply:
this is coming from our .net friend.
once the url is ready for dotnet this will be work too:
see dotnet: https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos.Encryption/src/CosmosEncryptor.cs#L13

Reply:
FYI @j82w @anujtoshniwal

Reply:
Sounds good then.